samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -1731,13 +1670,8 @@ void CEncodingPipeline::Close()
     MSDK_SAFE_DELETE(m_pmfxENC);
     MSDK_SAFE_DELETE(m_pmfxVPP);
 
-#if (MFX_VERSION >= 1024)
```

Comment:
Please revert it back. I assume it's necessary.

Reply:
Done